### ISSUE TITLE
(1.7.10) Firestone ore is missing from various menus, and appears to not be genning (9.12.2.0)

### ISSUE THREAD

**Issue by Trisscar05** [Railcraft/Railcraft#1355]
As the title, it's not appearing in NEI nor the creative menu, and a quick run through the nether (with sigil of suppression and speed buffs to cause the lava to temporarily become invisible as far as can be loaded) has revealed no ore blocks at all.
Natura is installed, though it was added a bit later than a bunch of the nether was loaded, so there is a giant part around the portal that contains no heat sand or tainted soil. Configs have nothing disabled and in fact firestone ore is mentioned and set to true in terms of ore gen, and presumably has been for as long as the pack has been installed since I never messed with it. Log has no mention of firestone at all, and nothing seems to have gone overly wrong with Railcraft's initialization as a whole.
Also, the various bits of non-ore firestone DO appear in NEI, but they seem to have no recipes, even counting each other, beyond a note about what enchantments they are valid with, making cut, refined, and charged firestone uncraftable even if I were to spawn the raw version in. MFR laser has no focus registered for the ore or the raw form (which tbf is likely intentional but thought I'd mention).

This is all part of a pack I took off Curse/Twitch and messed with a great deal to get it to where I wanted, so I'm not really looking for a one click solution. Mostly just trying to figure out what of all this is intended behavior, to guide what I do to try to fix it next. I can throw the log and such in as well if yall think they might be helpful, just felt like this wall of text was probably an adequate (and hopefully not overwhelming) start. XD

**Comment by liach** [comment]
@Trisscar05 Can you pull out that railcraft jar and play on a forge client with only that jar? If the problem persists, we will look into it.

**Comment by Trisscar05** [comment]
Pulling out the exact files and throwing them into a test forge pack had everything (including, I assume, the gen of firestone ore, didn't have any quick ways to check) seeming to work normally.
Going back into the convoluted pack and replacing the Railcraft jar didn't seem to fix anything, so looks like it's probably a mod conflict. Time to dupe the profile and start ripping mods out till it works I suppose, I'll let yall know if I find anything.

Edit: Alright, I ended up rebuilding the entire pack alphabetically while periodically loading it to see if I could catch which mod was causing the issue, but weirdly that didn't reproduce it. Since the problem has persisted through several restarts of both MC and my desktop, I can only assume that something has been overwritten or not installed correctly, except that makes no sense because I already tried removing and reinstalling Railcraft which I'd assume would have fixed anything like that.
I'mma try throwing my save into the newly built pack and see if the issue reoccurs.....Yup, now it's fine, as long as I use the new build. *throws up hands*
